docs/api/extras/helpers/GridHelper.html

```diff
@@ -18,25 +18,25 @@
 		<h2>Example</h2>
 
 		<code>var size = 10;
-		var step = 1;
+		var divs = 1;
 
-		var gridHelper = new THREE.GridHelper( size, step );
+		var gridHelper = new THREE.GridHelper( size, divs );

```

Comment:
`divisions` would be a better variable name, and 10 would be a more typical value.